assets/js/state/selectors/index.js

```diff
@@ -69,6 +69,11 @@ const enrichInstances = (instances, sapSystemId, state) =>
       };
     });
 
+export const getClusterOnHost = (hostId) => (state) => {
```

Comment:
We don't have a cluster on a host, rather a host part of a cluster.
I believe naming should reflect this.

What about `getHostCluster` or `getClusterByHost`?